packages/admin/src/Resources/Pages/ManageRecords.php

```diff
@@ -7,4 +7,5 @@ class ManageRecords extends ListRecords
     use ListRecords\Concerns\CanCreateRecords;
     use ListRecords\Concerns\CanDeleteRecords;
     use ListRecords\Concerns\CanEditRecords;
+    use ListRecords\Concerns\CanViewRecords;
```

Comment:
Don't think this should be a default trait IMO

Reply:
IMO, from a CRUD perspective you should be able to Create/Read/Update/Delete if you have the right to do so. So for me, this should be a default trait.